src/ArduPilotPlugin.cc

```diff
@@ -802,7 +797,7 @@ void gz::sim::systems::ArduPilotPlugin::LoadImuSensors(
 
 /////////////////////////////////////////////////
 void gz::sim::systems::ArduPilotPlugin::LoadGpsSensors(
-    sdf::ElementPtr /*_sdf*/,
+    const sdf::ElementPtr &/*_sdf*/,
     gz::sim::EntityComponentManager &/*_ecm*/)
 {
   /*
```

Comment:
I don't think we'll ever implement GPS sensors in the plugin so perhaps we could remove this and other commented code? I have experimented with using AP_DDS to add extra sensors - there is a lot more flexibility there and it doesn't involve adding further complexity to the SITL_JSON interface or opening additional ports. Different message types can be provided at custom rates, rather than require everything is sampled at the physics loop rate.